.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: python
+python:
+  - "2.7"
+
+install:
+  - pip install -r requirements.txt
+
+script:
+  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
+  - ansible-playbook --syntax-check playbooks/byo/config.yml
+  # TODO(rhcarvalho): update make ci to pick up these tests
+  - nosetests test/
```

Comment:
Ah, and the reason for that is that just `nosetests` + `make ci` would run 37 tests twice. The tests are quick, but installing their dependencies is not. So no need to wait for dependency installation twice.